Robus/src/msg_alloc.c

```diff
@@ -148,6 +148,11 @@ static inline void MsgAlloc_ValidDataIntegrity(void);
 void MsgAlloc_Init(memory_stats_t *memory_stats)
 {
     //******** Init global vars pointers **********
+    // Reinit ll_service id
+    for (uint8_t i = 0; i < ctx.ll_service_number; i++)
+    {
+        ctx.ll_service_table[i].id = DEFAULTID;
+    }
```

Comment:
In fact we want to do this every time we empty the memory. We can put it outside but we need to do it multiple times and it is risky that we will forget it. Imagine that previously it was into Port Manager that didnt make any sense...

Reply:
In general the context fields are reseted in msg alloc init and the low level services are part of the context. I dont see why we should create new functions in robus level and add more complexity in any way

Reply:
My point of view is that MsgAlloc is a library aims to deal with message buffer and only that.  The purpose of this lib is to make sure that all access to this shared buffer are done correctly.

Resetting services ID is not directly related to this. It's an architectural problem, not a detailed implementation one.

I think we should care of software architecture as much as we can because this is the element that ensures the reliability and scalability of the product on the long term.